src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CCDCaseRepository.java

```diff
@@ -32,19 +32,23 @@
 @Service("caseRepository")
 @ConditionalOnProperty(prefix = "core_case_data", name = "api.url")
 public class CCDCaseRepository implements CaseRepository {
-    private final CCDCaseApi ccdCaseApi;
-    private final CoreCaseDataService coreCaseDataService;
+    private CCDCaseApi ccdCaseApi;
+    private CoreCaseDataService coreCaseDataService;
```

Comment:
Please revert these back to `final` and have a constructor injection as before. I assume it's accidentally  committed.

Reply:
This is because there is a circular dependency of spring beans.

Reply:
We should fix that circular dependency rather than changing the injection here

Reply:
It’s likely we have a design problem here. Which bean it is complaining about?  let's try to redesign the components properly rather than changing the injection.